README.md

```diff
@@ -120,3 +130,32 @@ Be careful with this method of deployment. `bootRun` is not designed to work wit
 ## External
 
 Deploy resultant `cas/build/libs/cas.war` to a servlet container of choice.
+
+# DEBUG
+
+## Build
+
+```bash
+# shortcut for clean, build and explodeWar (mainly for debug purposes)
+./gradlew[.bat] buildit
```

Comment:
Gradle is not there to replace the bash from maven ? That is like a shorcut, I mainly use it in my IDE. It could be removed if not needed